### PR TITLE
Common - Add `FUNC(getItemReplacements)`

### PR DESCRIPTION
**Pull request by LinkIsGrim**
**When merged this pull request will:**
- Add an API function to get item replacements from an item type or classname.

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/common/functions/fnc_getItemReplacements.sqf [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * Return Value:
+ * Item replacements <ARRAY of STRING>
+ *
+ * Example:
+ * ["FirstAidKit", false] call ace_common_fnc_getItemReplacements
+ *
+ * Public: Yes
+ */
+
+params [["_item", "", [0,""]], ["_includeType", true, [true]]];
```
Comment (johnb432) [inline addons/common/functions/fnc_getItemReplacements.sqf:19]:
```suggestion
params [["_item", "", [0, ""]], ["_includeType", true, [true]]];
```

File: addons/common/functions/fnc_getItemReplacements.sqf [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * 0: Item or item type ID to check <STRING or NUMBER>
+ * 1: Include type replacements for items <BOOL> (default: true)
+ *
+ * Return Value:
+ * Item replacements <ARRAY of STRING>
+ *
+ * Example:
+ * ["FirstAidKit", false] call ace_common_fnc_getItemReplacements
+ *
+ * Public: Yes
```
Comment (johnb432) [inline addons/common/functions/fnc_getItemReplacements.sqf:16]: Are you sure you want to make this API?

Comment (johnb432) [reply addons/common/functions/fnc_getItemReplacements.sqf:16]: I'm just thinking of potential maintenance - up to you.

**Comment by johnb432** [comment]
You can register `[]` as an item replacer, which removes the registered item. If this is going to be API, we need a way to tell between that and the default value (maybe return `nil` if not defined?).

**Review by johnb432 — APPROVED** [review]

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**